client/images/p5js-logo-small.svg

```diff
@@ -1,13 +1,13 @@
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="56" height="42" viewBox="0 0 56 42">
     <defs>
-        <path id="a" d="M0 0h36v36H0z"/>
+        <path id="a" d="M1 0h34v35H1z"/>
         <path id="b" d="M0 0h36v36H0z"/>
     </defs>
     <g fill="none" fill-rule="evenodd">
         <path fill="#ED225D" d="M0 0h56v42H0z"/>
         <g transform="translate(10)">
             <use fill="#FFF" xlink:href="#a"/>
-            <path stroke="#979797" d="M.5.5h35v35H.5z"/>
+            <!-- <path stroke="#979797" d="M.5.5h35v35H.5z"/> -->
```

Comment:
Great catch!  This path absolutely does not need to be here! Instead of commenting it out, let's fully delete it. 🗑️